src/skmultiflow/meta/online_boosting.py

```diff
@@ -109,6 +108,7 @@ def __configure(self):
         self.lam_sc = np.zeros(self.actual_n_estimators)
         self.lam_sw = np.zeros(self.actual_n_estimators)
         self.epsilon = np.zeros(self.actual_n_estimators)
+        self._is_empty = True
```

Comment:
Is this variable used in other contexts?

Reply:
No, it should not be there. Removed in 0d7aaeeb2841b9e5dbfe80fb821691ee2fbbd166